common/data/src/main/java/org/thingsboard/server/common/data/device/profile/lwm2m/OtherConfiguration.java

```diff
@@ -30,6 +30,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class OtherConfiguration extends PowerSavingConfiguration {
 
+    private Boolean useObject19ForOta;
```

Comment:
useObject19ForOtaInfo

Reply:
Ok - useObject19ForOtaInfo + "Use Object 19 for OTA file metadata (checksum, size, version, name)"